### PR TITLE
Support null labels/parameters

### PR DESCRIPTION
**Pull request by jayunit100**
@jwendell support null / empty params

### CODE REVIEW

File: fabric8/src/main/java/org/jboss/arquillian/ce/fabric8/F8OpenShiftAdapter.java [hunk 1]
```diff
@@ -430,9 +430,14 @@ private boolean isBound(PersistentVolume pv, String targetClaimName, String targ
     private KubernetesList processTemplate(String templateURL, List<ParameterValue> values, Map<String, String> labels) throws IOException {
         try (InputStream stream = new URL(templateURL).openStream()) {
             ClientTemplateResource<Template, KubernetesList, DoneableTemplate> template = client.templates().inNamespace(configuration.getNamespace()).load(stream);
-            template.get().getLabels().putAll(labels);
+            if (template.get().getLabels() != null) {
+	         template.get().getLabels().putAll(labels);
+	     }
+	     if (values == null) {
```
Comment (jwendell) [inline fabric8/src/main/java/org/jboss/arquillian/ce/fabric8/F8OpenShiftAdapter.java]: values should never be null...

Comment (jayunit100) [inline fabric8/src/main/java/org/jboss/arquillian/ce/fabric8/F8OpenShiftAdapter.java]: hmm whys that? what if youre not parameterizing anything?

Comment (jwendell) [inline fabric8/src/main/java/org/jboss/arquillian/ce/fabric8/F8OpenShiftAdapter.java]: then values would be an empty List. But not null.

**Comment by jayunit100** [comment]
Updated,to explicit the non null part  as per @jwendell comments

**Comment by jwendell** [comment]
LGTM, although the @NotNull would be the first in the project, should we start using those annotations?

**Comment by jayunit100** [comment]
ok thanks for merging
why no NotNull ?

**Comment by alesj** [comment]
So far we haven't used it.
And imo, it's either all or nothing.
Where I don't see us now doing it "all".

**Pull request merged**